src/content/subgames.cpp

```diff
@@ -210,40 +222,46 @@ SubgameSpec findWorldSubgame(const std::string &world_path)
 	return findSubgame(world_gameid);
 }
 
-std::set<std::string> getAvailableGameIds()
+GamePathMap getAvailableGamePaths()
 {
-	std::set<std::string> gameids;
-	std::set<std::string> gamespaths;
-	gamespaths.insert(porting::path_share + DIR_DELIM + "games");
-	gamespaths.insert(porting::path_user + DIR_DELIM + "games");
+	GamePathMap gamepaths;
+	std::vector<GameFindPath> game_search_paths{
```

Comment:
could be an `std::array`

Reply:
No, because I'm adding stuff a few lines later (L236).